app/frontend/src/pages/chat/Chat.tsx

```diff
@@ -393,6 +415,33 @@ const Chat = () => {
                                         </div>
                                     </div>
                                 ))}
+                            {partialResponse && !isStreaming && (
+                                <div>
+                                    <UserChatMessage message={lastQuestionRef.current} />
+                                    <div className={styles.chatMessageGpt}>
+                                        <Answer
```

Comment:
Why is it that we need to have this if/else with a whole Answer component configuration, versus sending in a parameter to Answer?

Reply:
That's a good question. I believe I remember running into weird issues when dealing with the partial responses that would sometimes cause the last messages to disappear. This was put into place to circumvent that, but I am all ears if there is a cleaner implementation!